docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '2'
+services:
+  autopush:
+    image: bbangert/autopush
+    hostname: autopush
+    environment:
+      - LOCAL_HOSTNAME=localhost
+      - ROUTER_HOSTNAME=autopush
+      - HUMAN_LOGS=true
+      - CRYPTO_KEY=${CRYPTO_KEY}
+    ports:
+      - "8080:8080"
+      - "8081:8081"
+    links:
+      - dynamodb
+    volumes:
+      - ./boto-compose.cfg:/etc/boto.cfg:ro
+  autoendpoint:
+    image: bbangert/autopush
```

Comment:
Keep thinking we should move stuff like this under mozsvc. I'm presuming you like it here, but pointing off to a private repo is probably not a great practice.